### DISCUSSION TITLE
Updating realm config does not add idp

### DISCUSSION THREAD

**Discussion by jaymberry** [keycloak/keycloak#16706] [Archive]
I have config for our realm managed in github, but when I run the following, the identity provider in the realm.json does not get imported:
`./bin/kcadm.sh update realms/demorealm --realm master -f data/import/ealm.json --server http://keycloak-test:8080/auth`

Do I need to have the realm created first?

**Comment by Fikralaksana** [comment]
in my case, the realm exists but the IDP not being added as well.
